docs/StardustDocs/topics/overview.md

```diff
@@ -133,3 +133,13 @@ dfClean.groupBy { From into "origin" }.aggregate {
     pivot { To }.sum { recentDelays.intCols() } into "total delays to"
 }
 ```
+
+## Contribute and give feedback
```

Comment:
We have a contributing guide, please add it here and navigate users there 
https://github.com/Kotlin/dataframe/blob/master/CONTRIBUTING.md

Probably some information is repeated twice here and in this guide.

Reply:
The guide is quite big to add to the page. Perhaps instead we can create a separate page for it and link to it from the Overview? WDYT?

Reply:
Updated! If you're happy that this is resolved, please resolve this conversation.